README.md

```diff
@@ -1,2 +1,34 @@
 # storcycle_api
```

Comment:
Probably want to put in the link to the OpenAPI here with words.

Reply:
@LogicalChaos, what do you mean by "the OpenAPI"? A link to the Python examples file or to the swagger documentation?